lua/net.lua

```diff
@@ -1,17 +1,61 @@
 local gatekeeperc = require("gatekeeperc")
+local ffi = require("ffi")
+local ifaces = require("if_map")
 
 local M = {}
 
+function init_pci_addrs(iface, ports)
+	local pci_strs = ffi.new("const char *[" .. #ports .. "]")
+	for i, v in ipairs(ports) do
+		pci_strs[i - 1] = ifaces[v]
+	end
+	iface.pci_addrs = gatekeeperc.lua_init_pci_addrs(pci_strs, #ports)
+end
+
 -- Function that sets up the network.
 function M.setup_block()
 
 	-- Init the network configuration structure.
 	local conf = gatekeeperc.get_net_conf()
-	conf.num_rx_queues = 1
-	conf.num_tx_queues = 1
+
+	-- Change these parameters to configure the network.
+	local front_ports = {"enp131s0f0"}
+	local back_ports = {"enp133s0f0"}
+	conf.num_rx_queues = 2
+	conf.num_tx_queues = 2
+
+	-- Code below this point should not need to be changed.
+	conf.front.num_ports = #front_ports
+	conf.front.name = gatekeeperc.lua_init_iface_name("front")
```

Comment:
Thanks Cody for the clarification! The code looks simpler. Also, I have checked the APIs for getting the device information. Indeed, I found that the name mappings in DPDK are quite different from the kernel ones. So, I agree that it's necessary to obtain them before the network adapters are bound to DPDK.